diagnostic.md

```diff
@@ -6,72 +6,75 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The allow data to persist once a client's browser is closed and to allow users to access data from different clients.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+The model.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+The controller.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+Because they are no longer needed for the most part within the newer ansynchronous model.
 ```
 
 What does C.R.U.D stand for?
 
 ```bash
-// your response here
+Create, retrieve, update, delete.
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```bash
-// your response here
+In the controller (typically in the routes.rb file).
 ```
```

Comment:
You're routes file lists your routes thus sometimes rails is referred to as (R)MVC.  You are correct that CRUD actions are found in the controller but the controllers are in the controller folder and are typically named thins like userController, and peopleController.